Makefile

```diff
@@ -116,6 +116,9 @@ endif
 ifeq ("$(TEST_BENCHMARK)", "true")
 	TAGS := -bench=. -run=notests -cpu=1,2,4
 	TESTS := $(REPOPATH)/test
+ifndef TESTOPTIONS
```

Comment:
we have it already checked in the line 22 https://github.com/arangodb/go-driver/blob/b1d2682231f0a613c1a967261595ce0945ad47da/Makefile#L22-L22